From 2822a5ea109cc6177d37f4189f12a91d9e2b444f Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Wed, 2 Sep 2015 14:20:36 +0800
Subject: [PATCH] MLK-11489 i.MX6UL: Add the 9x9-evk-csi.dts for the camera
 module

This patch add the 9x9-evk-csi.dts for the camera module

since the csi-camera has the pin conflict with sim module,
this patch is to enable the camera module while disable sim.

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/boot/dts/Makefile               |  1 +
 arch/arm/boot/dts/imx6ul-9x9-evk-csi.dts | 22 ++++++++++++++++++++++
 2 files changed, 23 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index c76e71b..bc62ce0 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -230,6 +230,7 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx6ul-14x14-evk-csi.dtb \
 	imx6ul-9x9-evk.dtb \
 	imx6ul-9x9-evk-ldo.dtb \
+	imx6ul-9x9-evk-csi.dtb \
 	vf610-cosmic.dtb \
 	imx6sx-sdb-emmc.dtb \
 	imx6sx-sdb-m4.dtb \
diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk-csi.dts b/arch/arm/boot/dts/imx6ul-9x9-evk-csi.dts
new file mode 100644
index 0000000..2177807
--- /dev/null
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk-csi.dts
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6ul-9x9-evk.dts"
+
+
+&csi {
+	status = "okay";
+};
+
+&ov5640 {
+	status = "okay";
+};
+
+&sim2 {
+	status = "disabled";
+};
-- 
1.8.0

